runtime/js/40_performance.js

```diff
@@ -9,6 +9,8 @@
   const customInspect = Symbol.for("Deno.customInspect");
   let performanceEntries = [];
 
+  const timeOrigin = opNow();
+
```

Comment:
This isn't quite correct. Firstly, top level code is actually executed during snapshotting (during build). That means `timeOrigin` would be hardcoded to the time of build. Secondly `opNow` returns the time relative to the start of the program. `timeOrigin` is meant to be the timestamp since unix epoch.

Reply:
I suggest you call `new Date()` during `bootstrapRuntime()` in 99_main.js to get the time since unix epoch.